hw-2/package.json

```diff
@@ -9,17 +9,26 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "tsc": "tsc"
   },
+  "_moduleAliases": {
+    "@app-models": "build/models"
```

Comment:
Interesting! What's this for? Wasn't the change in `tsconfig.json` enough?

Reply:
Unfortunately it didn't work. 
Here's a separate PR where I enabled that feature: https://github.com/interist32/node.js-mentoring-program/pull/5